sections/03_categorize.md

```diff
@@ -63,6 +63,29 @@ subsequent training. Similar strategies have been employed for EHR data where
 high-quality labeled examples are also difficult to obtain
 [@doi:10.1101/039800].
 
+Chest X-ray image analysis is another area where recent DL methods showed
+remarkable results [@arXiv:1701.06599 @arXiv:XXX]. Alternative to using fully
+labeled training data, the authors in [@arXiv:XXX] proposed to use weakly
+labeled images for training deep learning models. To generate weak labels for
+x-ray images, Wang et al. [1] applied a series of Natural Language Processing
+(NLP) techniques to the associated chest X-ray radiological reports.
+Specifically, they first extracted all diseases mentioned in the reports using a
+state-of-the-art NLP tool, then applied a newly developed negation and
+uncertainty detection tool (NegBio) to filter negative and equivocal findings in
+the reports. Evaluation on three independent datasets demonstrated that NegBio
+is highly accurate for detecting negative and equivocal findings (~90% in
+F-measure overall). These highly accurate results meet the need to generate a
+corpus with weak labels, which serves as a solid foundation for the later
+process of image classification. The resulting dataset, known as CXR-XIV,
+consists of 108,948 frontal-view chest X-ray images (from 32,717 patients) and
+each image is associated with one or more weakly-labeled pathology category
+(e.g., pneumonia and cardiomegaly) or “normal” otherwise. Given such a large-
+scale dataset, Wang et al. [@arXiv:XXX] further proposed a unified weakly-
+supervised multi-label image classification framework, which involved using deep
+learning to effectively detect common thoracic diseases in radiology images and
+showed superior performance than those using a small set of fully labeled data
+in benchmarking results.
```

Comment:
So - it would be worth highlighting that the computationally augmented training dataset is very helpful here. This would tie into techniques to augment training datasets that come up a few times.

Also, will this dataset be available? If so, it would be good to highlight that in the discussion.